### ISSUE TITLE
val and test set contain train images from the constituting datasets?

### ISSUE THREAD

**Issue by svenkreiss** [Jeff-sjtu/CrowdPose#15]
Hello @Jeff-sjtu , thanks for providing the code and dataset. I am trying to produce numbers with OpenPifPaf on your CrowdPose dataset. When I take a COCO pretrained model, the test AP on CrowdPose is very good and I am trying to understand whether that is realistic. When you assembled your train/test split, did you keep the train/test separation from COCO?

What I am trying to understand is whether there are COCO train images in the CrowdPose val and test split which would distort my numbers.

Thanks for your help.

**Comment by ArchNew** [comment]
I have the same problem.

**Comment by TianpengBu** [comment]
The same concern here...
